editor/app/src/main/java/ca/ubc/eml/soiltopargraphy/editor/ui/infopanel/JInfoPanel.java

```diff
@@ -0,0 +1,50 @@
+package ca.ubc.eml.soiltopargraphy.editor.ui.infopanel;
+
+import ca.ubc.eml.soiltopargraphy.editor.ui.quizpanel.QuizPanel;
+
+public class JInfoPanel {
+
+    private String name;
+    private String description;
+    private String image;
```

Comment:
Instead of String for image type, use "Bitmap" to hold an image, which essentially stores a pixels array. Read more about Bitmap (https://developer.android.com/reference/android/graphics/Bitmap.